kolibri/plugins/style_guide/assets/src/views/content/button.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+
+  <div>
+    Button
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+  };
+
+</script>
+
+
+<style lang="stylus" scoped>
+
+  @require '~kolibri.styles.definitions'
```

Comment:
Thanks for the review!

This @require was added by Christian as part of the initial plugin setup. And during one of our meetings, we decided to use some sort of global CSS properties/variables to enforce consistent typography, spacing, color, etc. So we might actually need to refer to some of the core style definitions so we can show them in the style guide?